### PR TITLE
allow . - symbol in schemas

### PR DESCRIPTION
**Pull request by kahirokunn**
The swagger.yaml generated by the open api generator frequently contains . and - signs are frequently included in the swagger.yaml generated by the open api generator.

### CODE REVIEW

File: src/codegen/generate.ts [hunk 1]
```diff
@@ -242,8 +242,11 @@ export default class ApiGenerator {
     let ref = this.refs[$ref];
     if (!ref) {
       const schema = this.resolve<OpenAPIV3.SchemaObject>(obj);
+      const removeChars = ['\/', '.', '-']
```
Comment (Richard87) [inline src/codegen/generate.ts]: Could you add `:` as well?

Comment (kahirokunn) [reply src/codegen/generate.ts]: I've added it!

**Comment by kahirokunn** [comment]
@phryneas Hello?

**Comment by phryneas** [comment]
sorry. I'll try to take care of this tomorrow.

**Pull request merged**